Dockerfile

```diff
@@ -6,17 +6,14 @@
 FROM golang:1.14.3-alpine AS build
 RUN apk add git bash make
 RUN mkdir /app
+COPY go.mod go.sum /app/
 WORKDIR /app
-COPY go.mod .
-COPY go.sum .
 
 # Get dependencies - will also be cached if we won't change mod/sum
 RUN go mod download
 
-COPY cmds cmds
-COPY pkg pkg
-
-RUN go install ./...
+COPY . /app
```

Comment:
I don't think we want to copy everything -- that will mean the image will change any time anything changes (including, e.g., simply build configurations).  Copying cmds & pkg individually allows us to keep the image the same when nothing relevant has changed.

Reply:
I added a commit to address this.